test/LiquidDemocracy.test.js

```diff
@@ -10,53 +10,100 @@ const should = require('chai')
 
 contract('LiquidDemocracy', function (accounts){
 
-  beforeEach(async function () {
-    // Set up ballot instance and register 4 voters
-    this.liquidDemocracyBallot = await LiquidDemocracy.new();
-    for (let i = 0; i < 4; i++) {
-      await this.liquidDemocracyBallot.registerNewVoter({ from: accounts[i] });
-    }
-  });
-  
-  describe('registration set up', function () {
-    it('should register voters successfully', async function () {
-      const registered = await this.liquidDemocracyBallot.getVoterData(accounts[0]);
-      let registeredWeight = registered[0].toString();
-      let registeredStatus = registered[1];
-      const unregistered = await this.liquidDemocracyBallot.getVoterData(accounts[4]);
-      let unregisteredWeight = unregistered[0].toString();
-      let unregisteredStatus = unregistered[1];
-  
-      registeredWeight.should.equal('1');
-      registeredStatus.should.equal(true);
-      unregisteredWeight.should.equal('0');
-      unregisteredStatus.should.equal(false);
-    });
-  });
-  
-  describe('voting', function () {
-    it('should allow registered voters to vote', async function () {
-      await this.liquidDemocracyBallot.vote(accounts[0], 0);
-      await this.liquidDemocracyBallot.vote(accounts[1], 0);
-      await this.liquidDemocracyBallot.vote(accounts[2], 1);
-      const voteCountOption0 = await this.liquidDemocracyBallot.getBallotVoteCount(0);
-      const voteCountOption1 = await this.liquidDemocracyBallot.getBallotVoteCount(1);
-
-      voteCountOption0.toNumber().should.equal(2);
-      voteCountOption1.toNumber().should.equal(1);
+    beforeEach(async function () {
```

Comment:
Looks like the indentation space is mismatching what's in `master` for this file, was this intentional?

Reply:
Yes, indentation is different than the rest of the files, so I fixed it to standardize it

Reply:
Yea I was intentionally using 2 spaces for this .js file (because of personal preference) and 4 spaces for .sol files (because of the [style guide](https://solidity.readthedocs.io/en/v0.3.1/style-guide.html#indentation)). But yea maybe we should just be consistent with indentation spaces across files, so I'm fine with this, let's keep it 4-spaces all throughout